spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -169,23 +182,32 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
 		assertThat(latch.await(60, TimeUnit.SECONDS)).isTrue();
 		assertThat(rebalancePartitionsAssignedLatch.await(60, TimeUnit.SECONDS)).isTrue();
 		assertThat(rebalancePartitionsRevokedLatch.await(60, TimeUnit.SECONDS)).isTrue();
+		assertThat(listenerThreadNames).allMatch(new Predicate<String>() {
```

Comment:
See our `build.gradle`:
```
	compileJava {
		sourceCompatibility = 1.7
		targetCompatibility = 1.7
	}

	compileTestJava {
		sourceCompatibility = 1.8
		targetCompatibility = 1.8
	}
```

Actually my IDEA suggests me to convert it into Lambda anyway :wink: